examples/anvil/WORKSPACE

```diff
@@ -28,12 +28,9 @@ http_archive(
 
 load(
     "@build_bazel_rules_android//android:rules.bzl",
-    "android_ndk_repository",
```

Comment:
yass qween

Reply:
The ndk rules are a mess. Don't work past 27 afaict.